koku/masu/util/azure/common.py

```diff
@@ -186,10 +186,13 @@ def azure_generate_daily_data(data_frame):
     return data_frame
 
 
-def match_openshift_resources_and_labels(data_frame, cluster_topology, matched_tags):
+def match_openshift_resources_and_labels(data_frame, cluster_topologies, matched_tags):
     """Filter a dataframe to the subset that matches an OpenShift source."""
-    nodes = cluster_topology.get("nodes", [])
-    volumes = cluster_topology.get("persistent_volumes", [])
+    nodes = []
+    volumes = []
```

Comment:
The same technique could be done here.

Reply:
implemented the change to using chain from itertools, good suggestion.